library/network_connections.py

```diff
@@ -1029,6 +1029,7 @@ def connection_create(self, connections, idx, connection_current=None):
             addrs4 = list([a for a in ip["address"] if a["family"] == socket.AF_INET])
             addrs6 = list([a for a in ip["address"] if a["family"] == socket.AF_INET6])
 
+            is_ipv4_disabled = False
```

Comment:
regarding the commit message: It is not refactoring, if you change the behavior, IMHO. The change is something like "Set DNS search setting only for enabled IP protocols", then the details can explain the reasons for this.

Reply:
Thanks for the note. I updated.